Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -381,8 +381,8 @@ inline core::FieldFilter Filter(absl::string_view key,
 
 inline core::FieldFilter Filter(absl::string_view key,
                                 absl::string_view op,
-                                const std::string& value) {
-  return Filter(key, op, model::FieldValue::FromString(value));
+                                const char* value) {
```

Comment:
This fixes my test failures since string literals ended up creating boolean filters with the new boolean overload just below.